tests/simulator/test_initialization.py

```
@@ -347,117 +381,58 @@
-                def reshape(patch_data, nGhosts):
+                pdatas = patch.patch_datas
+                layout = pdatas["protons_Fx"].layout
+                centering = layout.centering["X"][pdatas["protons_Fx"].field_name]
+                nbrGhosts = layout.nbrGhosts(
+                    interp_order, centering
+                )  # primal in all directions
+                select = tuple([slice(nbrGhosts, -nbrGhosts) for i in range(dim)])
+
+                def domain(patch_data):
+                    if dim == 1:
+                        return patch_data.dataset[select]
```

Ruff

Function definition does not bind loop variable `select`

(B023)